behat.yml

```diff
@@ -3,7 +3,7 @@ default:
         Behat\Symfony2Extension:
         Behat\MinkExtension:
             default_session: selenium2
-            base_url: 'http://openorchestra.test:8000/app_test.php'
-            browser_name: chrome
+            base_url: 'http://admin.openorchestra.dev/app_behat.php'
```

Comment:
you should let the previous url, this way we can execute the behat test on the integration server